### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by HerrKartoffelchip**

### CODE REVIEW

File: accounts/views.py [hunk 1]
```diff
@@ -91,9 +91,13 @@ def view_profile(request):
         )
         p_form = ProfileUpdateForm(instance=request.user.profile)
     current_user = request.user
+    is_admin = False
```
Comment (SimonYSC) [inline accounts/views.py:94]: I don't think this is a good way to check if a user is admin or not. The superusers can have different account with different names. It's better to check if the current user is a superuser.

Reference:
https://stackoverflow.com/questions/65421561/how-can-i-check-if-an-user-is-superuser-in-django

**Review by SimonYSC — CHANGES_REQUESTED** [review]
Please make the change accordingly before merging.
- Checking if current user is a superuser using hard-coded variable will be hard to maintain.
- flag_comment feels like a page allowing people to flag others' comments, suggest changing to "flagged_comment"
- It's better to remove test printing function call and commented out code before pushing.

File: accounts/views.py (hunk 1 above)
Comment (HerrKartoffelchip) [reply accounts/views.py:94]: Good point, change applied!

File: naturescall/urls.py [hunk 2]
```diff
@@ -29,11 +29,8 @@
     ),
     path("coupon_register/<int:r_id>", views.coupon_register, name="coupon_register"),
     path("coupon_edit/<int:r_id>", views.coupon_edit, name="coupon_edit"),
-    path(
-        "flag_comment/<int:rating_id>",
-        views.flag_comment,
-        name="flag_comment",
-    ),
+    path("flag_comment/<int:rating_id>", views.flag_comment, name="flag_comment",),
```
[the remark this answers is not in the document]
Comment (HerrKartoffelchip) [reply naturescall/urls.py:32]: flag_comment() allows user to flag other users' comments that are inaccurate or inappropriate. I only changed the coding style to satisfy flake8.

[the remark this answers is not in the document]
Comment (SimonYSC) [reply naturescall/urls.py:32]: Gotcha

**Review by SimonYSC — APPROVED** [review]
LGTM

**Pull request merged**